---
layout: subpage
title: Orione
---

ORIONE is a web based platform to build complex, reproducible workflows for NGS microbiology data analysis. Based on Galaxy, an open source framework for data-intensive computational analysis and data integration, ORIONE has been the first available platform that supports the whole life cycle of microbiology research allowing microbiologist without any specific hardware or informatics skill to conduct data-intensive computational analyses from quality control to microbial gene annotation.  
ORIONE is in production state and accessible 24/7 with minimal downtime and is supported by a state-of-the-art supercomputing center.  
ORIONE has been used for NGS training courses attended by 100+ researchers from 20+ institutions. ORIONE NGS courses acquired the national ECM certification and are part of the Galaxy Training Network.  
These  resources are actively employed by cooperative research efforts between CRS4 and leading research institutions in the biosciences.  

### Accessibility information
ORIONE is accessible as free public service at [https://orione.crs4.it](https://orione.crs4.it).  The suggested mode of use of ORIONE is through registration or through a collaborative research agreement (CRA). However, anyone can use the server, with or without an account. Some limitations nevertheless exist, i.e., for unregistered users (5GB of disk quota and 2 maximum concurrent jobs) or registered users (100GB of disk quota and 10 maximum concurrent jobs).

### Publications

1. Orsini, M. et al. (2016). Bacterial genomics data analysis in the next-generation sequencing era. Data Mining Techniques for the Life Sciences (Second Edition), Springer, 06/2015, pages 407--422 Methods in Molecular Biology vol. 1415 - april 2016.  
DOI:10.1007/978-1-4939-3572-7_21
	
2. Cuccuru, G. et al. (2014). An Automated Infrastructure to Support High-troughput Bioinformatics. In Proceedings of. IEEE International Conference on High Performance Computing & Simulation , 07/2014, pages 600-607.  
DOI: 10.1109/HPCSim.2014.6903742.
	
3. Leo, S et al. (2014). BioBlend.objects: metacomputing with Galaxy. Bioinformatics (Oxford, England), 06/2014, 1–2.  
DOI: 10.1093/bioinformatics/btu386. 
	
4. Cuccuru, G. et al. (2014). Orione, a web-based framework for NGS analysis in microbiology. Bioinformatics (Oxford, England), 03/2014, 30(13),  1928–9.  
DOI: 10.1093/bioinformatics/btu135. 
    
